### PR TITLE
Redcap api add instructions for not pushing your API token to GH

### LINKED ISSUE #279: add tip about how to not push your API token to github
Suggestion for the REDCap API module, at the end where it shows how to get the R/python code from the API playground: The code generated by REDCap includes a line that saves your API token right there in your analysis file. If folks are using git/GH, they can easily forget that line is in there and version control their script with the secret included -- since truly deleting things from git and GH is so tricky/impossible, let's instead encourage them to save secrets in a separate file right from the beginning, i.e. in this module. 
That might mean an additional short section within each of the R and python sections, or perhaps it would make the most sense to have one additional section at the end covering both (mostly language-agnostic explanation/advice, and then some little snippets showing how to implement in R and python).

### PR DESCRIPTION
**Pull request by rosemm**
Fixes #279 
Please check my python solution in particular! I'm not sure environment variables are actually the most sensible approach here (although my googling around supports that). If there's a better way to do this, don't hesitate to suggest that.

### CODE REVIEW

File: using_redcap_api/using_redcap_api.md [hunk 1]
```diff
@@ -436,6 +436,300 @@ Click on "result" in the environment pane to open a file viewer.
...
+So `redcap_api_token` and `REDCAP_API_TOKEN` could be two different environment variables on a Unix computer, but they would refer to the same thing on a Windows computer!
+
+By convention, environment variables are usually written in all caps, but you can choose to use lower case if you prefer.
+
+</div>
+
+Create the .env file
+---
+
+The `.env` file is just a text file, so you can create it in any text editor you like.
```
Comment (leemc-data-ed) [inline using_redcap_api/using_redcap_api.md:509]: It's possible it isn't really necessary, but we could potentially add a little something about text editors vs. something like Word (which some folks might think of as a text editor.

**Review by leemc-data-ed — COMMENTED** [review]
I just had one little comment-- but it's not even totally necessary I don't think, just something to maybe consider. Otherwise lgtm!

File: using_redcap_api/using_redcap_api.md (hunk 1 above)
Comment (rosemm) [reply using_redcap_api/using_redcap_api.md:509]: Oh, very good call! Thank you!

**Comment by drelliche** [comment]
Non-content comment: @rosemm and @leemc-data-ed this might be a good moment to update all the highlight boxes, not just the ones near the information about where to store the API token. (I'm doing a quick check of issue https://github.com/arcus/education_modules/issues/242 and commenting on already open PRs)

**Comment by rosemm** [comment]
Oh yes, definitely! I didn't think to look back over the other highlight boxes for this module, but I'll do that now. Good catch!

**Comment by rosemm** [comment]
@leemc-data-ed would you mind taking a look again, and let me know if you think the text editor bit is clearer now?
I also updated the highlight boxes.

**Review by leemc-data-ed — APPROVED** [review]
Everything lgtm!

**Pull request merged**